src/prof_data.c

```diff
@@ -814,8 +946,16 @@ prof_dump_gctx(prof_dump_iter_arg_t *arg, prof_gctx_t *gctx,
 	    (opt_prof_accum && gctx->cnt_summed.accumobjs == 0)) {
 		assert(gctx->cnt_summed.curobjs == 0);
 		assert(gctx->cnt_summed.curbytes == 0);
+		/*
+		 * These asserts would not be correct -- see the comment on races
+		 * in prof.c
+		 * assert(gctx->cnt_summed.curobjs_unbiased == 0);
+		 * assert(gctx->cnt_summed.curbytes_unbiased == 0);
+		*/
 		assert(gctx->cnt_summed.accumobjs == 0);
+		assert(gctx->cnt_summed.accumobjs_shifted_unbiased == 0);
 		assert(gctx->cnt_summed.accumbytes == 0);
+		assert(gctx->cnt_summed.accumbytes_unbiased == 0);
```

Comment:
These ones I think should never fail; the accumbyte counters only ever get incremented. If they're nonzero, it means they've been incremented, which means we shouldn't be deleting the tctx. Added a comment.

Reply:
I don't quite follow. I think when the `gctx` corresponds to no live allocations, `accumbytes_unbiased` can be non-zero, because it's a sum of the potentially non-zero `accumbytes_unbiased`s of the `tctx`s (for which a `free()` call after `prof_reset()` might not exactly cancel out the previous value)?

Reply:
Sorry - I thought it was about the `curbytes`. You're right!